gpkit/constraints/bounded.py

```diff
@@ -0,0 +1,68 @@
+"Implements BoundedConstraintSet"
+from collections import defaultdict
+import numpy as np
+
+from .. import Variable
+from .set import ConstraintSet
+from ..small_scripts import mag
+
+
+def add_bounds(varkeys, lower, upper):
+    "Returns constraints list bounding all varkeys"
+    constraints = []
+    for varkey in varkeys:
+        units = varkey.descr.get("units", 1)
+        constraints.append([upper*units >= Variable(**varkey.descr),
+                            Variable(**varkey.descr) >= lower*units])
```

Comment:
Does it make sense to make `Variable(**varkey.descr)` once and then use it instead of making two Variable objects here?